src/main/java/org/spongepowered/common/service/scheduler/AsyncScheduler.java

```diff
@@ -106,608 +51,139 @@
     private final Condition condition = this.lock.newCondition();
     // The dynamic thread pooling executor of asynchronous tasks.
     private ExecutorService executor;
-    // Query actor for task information
-    private SchedulerHelper schedulerHelper;
 
     private AsyncScheduler() {
```

Comment:
```
final Task task = this.game.getTaskBuilder()
    .async()
    .execute(new Runnable() {})
    .delay(10, TimeUnit.MILISECONDS)
    .repeatEvery(10, TimeUnit.MILISECONDS)
    .side(Platform.Side.SERVER) // Once Sponge is on the client, this can determine if to run on client or not
    .build()

this.game.getScheduler().submit(this, task);
```

I removed the setting of the boolean value and instead made it a toggle to on. I also included Kashike's suggestion.

Maybe roll this up into an API Scheduler cleanup @simon816 ?